### PR TITLE
docs: add the example for movable table

### PR DESCRIPTION
**Pull request by tinaClin**
## Row reordering with react-movable
image

## Row reordering with react-dnd
image

## Column reordering
image

## Resizable table
image

### CODE REVIEW

**Comment by roth1002** [comment]
Movable是取自哪裡的啊？

**Review by roth1002 — APPROVED** [review]

**Comment by tinaClin** [comment]
> Movable是取自哪裡的啊？

直接參考`react-movable`的~XD

**Comment by cheton** [comment]
> > Movable是取自哪裡的啊？
> 
> 直接參考`react-movable`的~XD

image

File: packages/styled-ui-docs/pages/table.mdx [hunk 1]
```diff
@@ -726,6 +726,77 @@ function FullWidthTable() {
 render(<FullWidthTable />);
 ```
 
+### Movable table
```
Comment (cheton) [inline packages/styled-ui-docs/pages/table.mdx]: `Movable table` 可以改名成 `Movable rows` or `Movable table rows`

**Comment by cheton** [comment]
@tinaClin 
Some feature requests to enrich the content of table examples

* Column dragging
https://codesandbox.io/s/gargroh-v7-drag-column-beautiful-react-dnd-encww

* Column resizing
https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/column-resizing

**Review by cheton — APPROVED** [review]

**Review by austinhung81 — APPROVED** [review]

**Pull request merged**